bitrise.yml

```
@@ -0,0 +1,65 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  check:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git: { }
+
+  e2e:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git@main: { }
+
+  sample:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/Bitrise-Android-Modules-Sample.git
+    - TEST_APP_BRANCH: main
+    - TEST_APP_MODULE: app
+    - TEST_APP_VARIANT: debug
+    steps:
+    - script:
+        inputs:
+        - content: |
+            #!/bin/env bash
+            set -euo pipefail
+            rm -rf "./_tmp"
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone:
+        inputs:
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - install-missing-android-tools:
+        run_if: .IsCI
+        inputs:
+        - gradlew_path: ./_tmp/gradlew
+    - git::https://github.com/bitrise-steplib/bitrise-step-android-build-for-ui-testing:
```

```suggestion
    - android-build-for-ui-testing:
```

In the e2e tests, we tend to use the latest version of the Steps from the StepLibrary. Usually, this version is what we have on the master branches of the Step repos, but still, the StepLib reference is more close to how customers are using Bitrise Steps.